x-pack/plugins/apm/common/elasticsearch_fieldnames.ts

```diff
@@ -7,7 +7,13 @@
 export const SERVICE_NAME = 'service.name';
 export const SERVICE_ENVIRONMENT = 'service.environment';
 export const SERVICE_AGENT_NAME = 'agent.name';
+export const SERVICE_AGENT_VERSION = 'agent.version';
```

Comment:
I think `SERVICE_AGENT_NAME` and `SERVICE_AGENT_VERSION ` should be called `AGENT_NAME` and `AGENT_VERSION` since they are no longer on the "service" context (they used to be afair)

Reply:
ah yeah was wondering why that was the case. I'll update